OrbitCore/StringManager.cpp

```diff
@@ -13,7 +13,14 @@ bool StringManager::AddIfNotPresent(uint64_t key, std::string_view str) {
   return true;
 }
 
-std::optional<std::string> StringManager::Get(uint64_t key) {
+bool StringManager::AddOrReplace(uint64_t key, std::string_view str) {
```

Comment:
The return value is not that clear from the method name. The caller needs to look at the implementation to know if `true` means "added" or "replaced". Not too sure what a better name would be though...

Reply:
Sounds good.